ground/src/main/res/layout/multiple_choice_checkbox_item.xml

```diff
@@ -42,16 +39,18 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:checked="@{item.isSelected}"
+      android:fontFamily="@font/google_sans"
       android:onClickListener="@{() -> viewModel.toggleItem(item)}"
       android:padding="16dp"
       android:text="@{ item.isOtherOption ? @string/other : item.option.label }"
+      android:textColor="@color/md_theme_onSurface"
+      android:textSize="16sp"
```

Comment:
Could you define this in `styles.xml` instead using the Material theme names from Figma here and throughout?  I think you need to extend `Widget.Material3.CompoundButton.CheckBox`.

Reply:
Obsolete since the UI has been migrated to compose.